src/axolotl/utils/callbacks.py

```diff
@@ -30,3 +31,39 @@ def on_save(
         kwargs["model"].save_pretrained(peft_model_path)
 
         return control
+
+
+class SaveBetterTransformerModelCallback(
+    TrainerCallback
+):  # pylint: disable=too-few-public-methods
+    """Callback to save the BetterTransformer wrapped model"""
+
+    def on_step_end(
```

Comment:
Maybe you should change to `on_save` like above. Then clean the save checks below.
```suggestion
    def on_save(
```

Edit: After reading your comments below. I'm not sure if this suggestion is correct.